src/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -50,6 +50,21 @@ internal PublicClientApplicationBuilder WithUserTokenLegacyCachePersistenceForTe
             return this;
         }
 
+        /// <summary>
+        /// Sets the public client application to use the latest default redirect URI as opposed to urn:ietf:wg:oauth:2.0:oob. 
```

Comment:
Needs review, I don't think users need to understand "latest" and the fact that in the past we were using urn:ieft...

I would reword it like "Configures the pca to use the recommended redirect Uri for the platform. If you are calling your API from a .NET Classic app - https://login.microsoftonline.com/common/oauth2/nativeclient, ....."

Reply:
@jmprieur any thoughts on what this comment should be?